src/mode/modeHandler.ts

```diff
@@ -555,9 +581,7 @@ export class ModeHandler implements vscode.Disposable {
       }
     }
 
-    if (ranAction && vimState.currentMode !== Mode.Insert) {
```

Comment:
Why was this needed? From what I've tested this is not necessary. And for this new remapper we need that every key that has been handled gets cleared from the commandList.